tests/scenarios/scenario1/pool-clear/3.check-node-pool.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const testUtils = require('../../../../lib/utils/test-utils')
+const utils = require('./utils')
+
+describe('New wallet which was sent transaction', () => {
+  it('should have coins in the wallet', async () => {
+    const response = await testUtils.request('GET', 'transactions/unconfirmed')
+    testUtils.expectSuccessful(response)
+    const transactions = response.data.data
+    
+    expect(transactions.length).toBe(0) // transaction was removed from pool
```

Comment:
1. The transaction was properly saved on disk during shutdown, but during startup was removed because other nodes told us it is already forged. => It is not in the pool.
2. The transaction was never saved on disk or it was saved but the startup code which is supposed to load all saved transactions does not work. => It is not in the pool.

In order to distinguish 1. from 2. we should have one more transaction which does not get forged while the node is down. That is - shutdown with 2 transactions, get one of them forged by others, start and observe that the other transaction is still in the pool, ie. it was properly saved and survived the restart.

Reply:
done in new version